man/crypttab.xml

```diff
@@ -260,6 +260,29 @@
         <xi:include href="version-info.xml" xpointer="v243"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>link-volume-key-to-keyring=</option></term>
+
+        <listitem><para>Specifies the keyring and key description where
+        LUKS2 volume key gets linked during device activation. The keyring
+        description and key description must be separated by <literal>::</literal>.</para>
+
+        <para>The keyring part can be a string description, or predefined
+        keyring prefixed with <literal>@</literal> (e.g.: to use @s session or
+        @u user keyring directly). The type prefix text keyring description is not required.</para>
+
+        <para>The key part is a string description optionally prefixed by a <literal>%key_type:</literal>.
+        If no type is specified, libcryptsetup uses <literal>user</literal> type key by default. See
```

Comment:
i think the word "libcryptsetup" should not be mentioned here. We typically do not mention the backend things are implemented in.